compiler/dyno/tools/chpldoc/chpldoc.cpp

```diff
@@ -1318,6 +1318,11 @@ struct RstResultBuilder {
   static const int commentIndent = 3;
   int indentDepth_ = 1;
 
+  RstResultBuilder(Context* context) {
```

Comment:
Why not just use:

```cpp
RstResultBuilder(Context* context) : context_(context), os_(std::stringstream()) {}
```

Reply:
that works too, I'm not sure what the difference is between the them though

Reply:
The second form will initialize each field only once and functions similarly to `in` intent formals in Chapel. Your form will default initialize any fields and then perform the assignment. In practice, probably doesn't matter, but I think the second form is a bit more idiomatic.